### ISSUE TITLE
When run Karabiner-Elements on macOS 10.12 , the scale screen feature be disabled

### ISSUE THREAD

**Issue by finscn** [tekezo/Karabiner-Elements#388]
When run Karabiner-Elements on macOS 10.12 , the scale screen feature be disabled.

(the scale screen feature used :  ctrl+ two fingers drag on trackpad )

If unchecked keyboard/trackpad in Devices tap of  Karabiner-Elements preferences ,  the feature will come back.

Device: MacBook Pro 2015
OS: macOS 10.12.1

**Comment by awenner** [comment]
Same problem occurs for me too. Karabiner-elements disables pinch-to-zoom on my trackpad.

**Comment by dunn** [comment]
It would be nice if the developers acknowledged this long-standing and serious issue impacting Accessibility.

**Comment by cranmer** [comment]
I have this problem too. It was fine with Karibiner on 10.11. @dunn I'm not sure calling this a long-standing issue is fair since this worked just fine prior to macOS 10.12 and they are clearly working to fix Karabiner given the OS changes.

**Comment by dunn** [comment]
@cranmer you're right.  I was just frustrated since this seems to have been present since Karabiner-Elements was released and it'd be nice to just hear 'we know, and we're working on it.'

**Comment by dvdplm** [comment]
I have this problem too. Would love to see a fix! :)

(and thank you so much for your hard work, I can't stress how much this software is helping me cope with an "odd" keyboard!)

**Comment by wesbos** [comment]
Same problem here - anyone know a fix?

**Comment by jaymzcd** [comment]
Ah I thought I was going crazy. I have the same problem. If I have 'Elements running then `ctrl+scroll` to zoom does not work, or at some times it seems like Karabiner will "stick" the `ctrl` key so that _any_ scrolling via the trackpad causes the whole screen to zoom. In other cases it will completely ignore the action.

I tried changing the `ctrl` key to be `option` or `cmd` in accessibility but no change. I also tried removing all key mappings in Karabiner and also mapping left ctrl to left ctrl  - no change in the broken behaviour.
